app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert split_integer(8, 1) == [8]
```

Comment:
You should test that the `sum` of parts equals to the initial value. Rewrite this test to obtain a list of several values and then apply `sum` to it, on the right side leave the required integer, not list. Your test body should contain one line with an `assert` statement as well.